prov/rxm/src/rxm_cq.c

```diff
@@ -964,8 +975,10 @@ ssize_t rxm_rndv_send_wr_data(struct rxm_rx_buf *rx_buf)
 			  rx_buf->recv_entry->rxm_iov.iov,
 			  rx_buf->recv_entry->rxm_iov.count, rx_buf->mr);
 
-	ret = fi_send(rx_buf->conn->msg_ep, &buf->pkt, sizeof(buf->pkt) +
-		      sizeof(struct rxm_rndv_hdr), buf->hdr.desc, 0, rx_buf);
+	ret = rxm_check_duplex_conn(&rx_buf->conn->handle);
```

Comment:
Similar question as above.  If the connection isn't ready, I would have expected a failure sooner than this call.